RectifyImage: True
show_ROI:False
CameraType: fisheye
CameraTopic: /lucid_driver/image
LidarTopic: /livox/lidar
CameraMatrix:
1178.0830595976777 0.0 1446.3220434510556
0.0 1178.03392240546 289.7313106674044
0.0 0.0 1.0
DistCoeffs: 0.1159621739356385 0.039360730097557625 0.04622367099013344 0.04656206896484101
ExtrinsicMat:
-0.00473265 0.999984 0.00294718 0.00801773
-0.00234125 -0.00295829 0.999993 0.0341689
0.999986 0.00472571 0.00235522 0.105932
0 0 0 1
